configuration/config.xml.tpl

```diff
@@ -32,6 +32,7 @@
     <permission>hudson.model.Item.Workspace:authenticated</permission>
     <permission>hudson.model.View.Read:authenticated</permission>
     <permission>jenkins.metrics.api.Metrics.View:authenticated</permission>
+    <permission>hudson.model.Hudson.Read:anonymous</permission>
```

Comment:
Seems reasonable. I might give this a spin later on, but not on wendy like I did earlier on :-/. Since we didn't add any way of updating this config, we will need to figure out how to migrate from one to the other (GlobalMatrixAuthorizationStrategy to ProjectMatrixAuthorizationStrategy) on a running instance.